app/helpers/session_helper.rb

```diff
@@ -3,8 +3,8 @@ def show_session(question)
     session[:registration][question] ? session[:registration][question].titleize : nil
   end 
 
-  def show_link(href)
-    "<a href='#{href}'>Change</a>".html_safe
+  def show_link(step)
```

Comment:
This module is missing tests

Reply:
Have put some basic tests up, the module will probably change following the content review, as the check answers page has to be changed to a GOV format